docs/entity-editor.rst

```diff
@@ -0,0 +1,59 @@
+######################
+The Entity Editor
+######################
+
+Introduction
+==============
+
+In Bookbrainz, a set of information about a particular author, book, publication, magazine, etc. is referred to as an *Entity*, and the form which we use to add/edit this information is called the **Entity-Editor**.
+We currently have the following types of entities in Bookbrainz:
+
+* Author
+* Work
+* Edition
+* Edition-Group
+* Series 
+* Publisher
```

Comment:
I would use this (which has a link to the page where we describe more) which I copied from another page:
```
There are 6 entity types in the BookBrainz world: Author, Work, Edition Group, Edition, Publisher and Series.
For a better description of each entity see :ref:`entities-description`.
```

Reply:
Makes sense. I have made the changes, as suggested.